tests/integration/expected_test_results/api_v2_product_read/get-existing-product.json

```diff
@@ -766,10 +766,10 @@
       "nutrient_levels" : {},
       "nutrient_levels_tags" : [],
       "nutriments" : {
-         "fruits-vegetables-legumes-estimate-from-ingredients_100g" : 25,
-         "fruits-vegetables-legumes-estimate-from-ingredients_serving" : 25,
-         "fruits-vegetables-nuts-estimate-from-ingredients_100g" : 25,
-         "fruits-vegetables-nuts-estimate-from-ingredients_serving" : 25,
+         "fruits-vegetables-legumes-estimate-from-ingredients_100g" : 62.5,
```

Comment:
It's changed because before we were computing the minimum estimate of fruits and vegetables, which was not realistic at all.

In this case, the ingredient list is "apple, milk, eggs, palm oil". So technically, there could be exactly 25% of each. Previously we took that as the estimate, it was the most conservative estimate.

Now we take our best estimate (62.5%, which is the middle of the possible range (it could be anywhere between 25% and 100%)). In the future, we should have better estimates (using nutrition facts + categories distributions of nutrients)